q2_assembly/quast/quast.py

```diff
@@ -54,10 +55,9 @@ def _process_quast_arg(arg_key, arg_val):
     """
     if isinstance(arg_val, bool) and arg_val:
         return [_construct_param(arg_key)]
-    elif arg_key == "threads" and (not arg_val or arg_val > 1):
-        # TODO: this needs to be fixed (to allow multiprocessing)
+    elif arg_key == "threads" and arg_val > 1 and platform.system() == "Darwin":
         print(
-            "Multiprocessing is currently not supported. Resetting "
+            "Multiprocessing is currently not supported on macOS. Resetting "
```

Comment:
I would error out here, so it's more clear to the user that something went wrong, and so they have the option to change course. It's possible they won't see this depending on the interface they're using. 

Also, in the spirit of my previous comment, I suggest changing this to "... only supported on Linux." That also gives them a clear direction to go next.